cli_tools/gce_vm_image_import/importer/args.go

```diff
@@ -223,6 +224,9 @@ func (args *ImportArguments) registerFlags(flagSet *flag.FlagSet) {
 	flagSet.Var((*flags.TrimmedString)(&args.ComputeEndpoint), "compute_endpoint_override",
 		"API endpoint to override default.")
 
+	flagSet.Var((*flags.TrimmedString)(&args.ComputeServiceAccount), "compute_service_account",
+		"Compute service account to override default.")
```

Comment:
```suggestion
	flagSet.Var((*flags.TrimmedString)(&args.ComputeServiceAccount), "compute_service_account",
		"Compute service account to be used by exporter Virtual Machine.")
```

Reply:
let me remove it for now since I only work on export...